src/rabbit_lvc_plugin.erl

```diff
@@ -19,7 +19,7 @@ setup_schema() ->
                         [{attributes, record_info(fields, cached)},
                          {record_name, cached},
                          {type, set}]),
-    mnesia:add_table_copy(?LVC_TABLE, node(), ram_copies),
+    mnesia:change_table_copy_type(?LVC_TABLE, node(), disc_copies),
```

Comment:
Would it work to add `{disc_copies, node()}`  to the table definition?

https://erldocs.com/current/mnesia/mnesia.html?i=0&search=mnesia:create_table#create_table/2